js/script.js

```diff
@@ -2,59 +2,45 @@ const toDoListHTML = document.querySelector('.toDoList__list');
 const addForm = document.querySelector('form.add');
 const addInput = addForm.querySelector('.adding__input');
 
-const getFilms = () => {
-  try {
-    return JSON.parse(localStorage.getItem('movies'));
-  } catch (error) {
-    setFilms([]);
-    return [];
-  }
-};
-
-const setFilms = (movies) => localStorage.setItem('movies', JSON.stringify(movies));
+const getStrings = () => JSON.parse(localStorage.getItem('list'));
 
-function checkLocalStorage() {
-  const movies = getFilms();
-
-  if (!movies && !Array.isArray(movies)) {
-    setFilms([]);
-  }
-}
-checkLocalStorage();
+const setStrings = (stringOfList) => localStorage.setItem('list', JSON.stringify(stringOfList));
 
-createMovieList();
+createToDoList();
 
 addForm.addEventListener('submit', (event) => {
   event.preventDefault();
 
-  let newMovie = addInput.value;
-
-  if (newMovie) {
-    const movies = getFilms();
-
-    if (newMovie.length > 21) {
-      newMovie = `${newMovie.substring(0, 22)}...`;
+  let newString = addInput.value;
+
+  if (newString) {
+    const stringOfList = getStrings();
+    if (
+      (!stringOfList && !Array.isArray(stringOfList)) ||
+      !Array.isArray(JSON.parse(localStorage.getItem('list')))

```

Comment:
Why do yo use this one `JSON.parse(localStorage.getItem('list'))` if you have this function `const getStrings = () => JSON.parse(localStorage.getItem('list'));` ?